adapters/repos/db/index.go

```diff
@@ -2054,6 +2054,32 @@ func (i *Index) initLocalShardWithForcedLoading(ctx context.Context, class *mode
 	return nil
 }
 
+func (i *Index) UnloadLocalShard(ctx context.Context, shardName string) error {
```

Comment:
Are there other places that `LoadAndDelete` then `Shutdown` that we can replace with this as well?

Reply:
i love this. `ShutdownShard()` on the Migrator could use this given this would be on index level